src/pages/library/DataBrowserPreview.js

```diff
@@ -19,8 +19,20 @@ const styles = {
   table: {
     header: {
       color: colors.accent(),
-      height: '1rem',
+      height: '2rem', lineHeight: '2rem',
       textTransform: 'uppercase', fontWeight: 600, fontSize: '0.75rem'
+    },
+    headerRow: {
+      borderTop: '1px solid rgba(0,0,0,.3)', borderBottom: '1px solid rgba(0,0,0,.3)'
```

Comment:
Note to self: We are inconsistent about "0.3" vs. ".3" for decimals in our styles. Might be worth a cleanup PR to have the same style everywhere.

Reply:
The standard in the codebase is `0.x`, so we should use that.